ckanext/canada/public/static/js/more_like_this.js

```diff
@@ -0,0 +1,13 @@
+
+$( document ).on( "wb-ready.wb", function( event ) {
+    $('#related_pkgs ul').removeClass('list-unstyled');
+    $('#related_pkgs li:gt(4)').remove();
+
+    var listItems = $("#related_pkgs a");
+    listItems.each(function(idx, a) {
+        var text = $(a).text().trim();
+        if (text.length > 180) {
+            $(a).text(text.substring(0,175) + '...');
+        }
+    });
+});

```

Comment:
@thriuin can this be done on the advanced search side? I don't like shortening content in JS on the client side like this.

Reply:
Sure, you can download the work to search instead of CKAN ;-) It means CKAN formatting logic is in search - why do you not like it in JS?

Reply:
It looks like this code is trimming the text length returned from the more like this entries after they are AJAX'ed in to the page. Isn't it simpler to trim them in the advanced search view and avoid the extra JS here?

Reply:
I suppose - it's not a lot of code either way. It's not a lot of extra text transmitted either I don't think, but I don't feel strongly either way. I can make the change - it's too small to open a ticket over.

Reply:
Thanks Ross, every bit of static JS we can keep out of this extension is going to make it simpler when we're migrating to the new way of serving static files in ckan 2.8/2.9

Reply:
Ah, makes sense for sure. No problem, it's just a matter of adding a filter to the Django template. Changes to search are being deployed to staging now. @HoussamBedja, you can remove this and let search truncate titles to a max of 175 chars.